### PR TITLE
raxml 8.2.11

### PR DESCRIPTION
**Pull request by jonchang**
- [x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

File: Formula/raxml.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Raxml < Formula
+  # cite: Stamatakis_2014: "http://doi.org/10.1093/bioinformatics/btu033"
+  desc "Maximum likelihood analysis of large phylogenies"
+  homepage "https://sco.h-its.org/exelixis/web/software/raxml/index.html"
+  url "https://github.com/stamatak/standard-RAxML/archive/v8.2.11.tar.gz"
+  sha256 "08cda74bf61b90eb09c229e39b1121c6d95caf182708e8745bd69d02848574d7"
+  head "https://github.com/stamatak/standard-RAxML.git"
+
+  depends_on "gcc"
```
Comment (sjackman) [inline Formula/raxml.rb]: Why does this formula depend on `gcc`?

File: Formula/raxml.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  # Won't build on OS X - relies on Linux-specific threading APIs
+  depends_on "open-mpi" => [:cc, :recommended] if OS.linux?
```
Comment (sjackman) [inline Formula/raxml.rb]: Change to…
```ruby
depends_on "open-mpi" => :recommended unless OS.mac?
```

File: Formula/raxml.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  def install
+    stems = %w[SSE3 AVX AVX2 SSE3.PTHREADS AVX.PTHREADS AVX2.PTHREADS]
+    stems << %w[SSE3.MPI SSE3.HYBRID AVX.MPI AVX.HYBRID AVX2.MPI AVX2.HYBRID] if build.with? "mpi"
```
Comment (sjackman) [inline Formula/raxml.rb]: I think you mean `+=` (concatenate) here rather than `<<` (append one item).

File: Formula/raxml.rb [hunk 4]
```diff
@@ -0,0 +1,39 @@
+class Raxml < Formula
+  # cite: Stamatakis_2014: "http://doi.org/10.1093/bioinformatics/btu033"
```
Comment (sjackman) [inline Formula/raxml.rb]: No colon `:` after cite.
```
cite Stamatakis_2014: …
```

**Comment by jonchang** [comment]
Thanks @sjackman! I've made the changes requested.

**Comment by jonchang** [comment]
Looks like the macOS builder doesn't support AVX etc instructions which is causing the build to fail.

**Comment by jonchang** [comment]
Builds 🍏 !

**Pull request closed without merging**